x-pack/plugins/canvas/canvas_plugin_src/uis/datasources/timelion.js

```diff
@@ -68,22 +90,48 @@ const TimelionDatasource = ({ args, updateArgs }) => {
         // TODO: Time timelion interval picker should be a drop down
       }
       <EuiFormRow
-        label="Interval"
-        helpText="Accepts Elasticsearch date math: 1w, 5d, 10s, or auto"
+        label={
+          <FormattedMessage
+            id="xpack.canvas.uis.datasource.timelionFormRowIntervalLabel"
+            defaultMessage="Interval"
+          />
+        }
+        helpText={
+          <FormattedMessage
+            id="xpack.canvas.uis.datasource.timelionFormRowIntervalHelpText"
+            defaultMessage="Accepts Elasticsearch date math: 1w, 5d, 10s, or auto"
+          />
+        }
         compressed
       >
         <EuiFieldText value={getInterval()} onChange={e => setArg('interval', e.target.value)} />
       </EuiFormRow>
 
-      <EuiCallOut color="warning" title="Some tips" size="s">
+      <EuiCallOut
+        color="warning"
+        title={
+          <FormattedMessage
+            id="xpack.canvas.uis.datasource.timelionTipsTitle"
+            defaultMessage="Some tips"
+          />
+        }
+        size="s"
+      >
         <ul>
           <li>
-            Timelion requires a time range, you should add a time filter element to your page
-            somewhere, or use the code editor to pass in a time filter.
+            <FormattedMessage
+              id="xpack.canvas.uis.datasource.timelionFirstTipDescription"
```

Comment:
do not call ids like first, second etc. for example this one I would call like howToAddTimeRangeDescription